src/components/Table/Table.jsx

```diff
@@ -116,6 +116,7 @@ const propTypes = {
       ),
       /** what is the current state of the row actions */
       rowActions: RowActionsStatePropTypes,
+      rowActionColumn: PropTypes.shape({ minWidth: PropTypes.string }),
```

Comment:
```suggestion

```

Reply:
fixed

Reply:
I'm still seeing these present in the diff?

Reply:
Hmm.. not sure how that got back in there. Should be gone now. Thanks :-)